src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -208,7 +208,6 @@ foam.CLASS({
           location.reload();
         } else {
           // logout once we have finished updating documents.
-          this.auth.logout();
```

Comment:
You should remove the comment on the previous line that says you're logging out, if you're no longer logging out.